shepherd_bms/src/compute.cpp

```diff
@@ -271,6 +271,29 @@ void ComputeInterface::sendCurrentsStatus(uint16_t discharge, uint16_t charge, u
     sendMessageCAN1(0x06, 8, currentsStatusMsg.msg);
 }
 
+void ComputeInterface::sendFaultStatus(BMSFault_t fault_status)
+{
+    static union
+    {
+        uint8_t msg[4] = {0, 0, 0, 0};
+
+        struct 
+        {
+            uint8_t faults_byte_3;
```

Comment:
I think you can actually declare just a uint32_t fault and then the union will take care of transposing it to the bytes.